libs/qhexedit/src/qhexedit.cpp

```diff
@@ -114,7 +114,7 @@ int QHexEdit::addressWidth()
     if (size > Q_INT64_C(0x100000000)){ n += 8; size /= Q_INT64_C(0x100000000);}
     if (size > 0x10000){ n += 4; size /= 0x10000;}
     if (size > 0x100){ n += 2; size /= 0x100;}
-    if (size > 0x10){ n += 1; size /= 0x10;}
+    if (size > 0x10){ n += 1;}
```

Comment:
Shouldn't this be requested upstream in the [qhexedit library](https://github.com/Simsys/qhexedit2), so it is not lost when we update to a new release?

Reply:
Ups, sorry, didn't pay attention. Will revert them.

Reply:
https://github.com/Simsys/qhexedit2/pull/105

Hope it's the correct repo 😆